blog/why-theres-no-new-episode-this-week.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Why There's No New Episode This Week
+publish: 2020-06-01T10:19:30
+slug: why-theres-no-new-episode-this-week
+---
+
+Due to current events happening in the United States regarding George Floyd and the protests to bring about changes to systemic racial injustice and police brutality, we will not be releasing a new episode this week and will be donating our sponsorship profits to [Campaign Zero](https://www.joincampaignzero.org/) from last week's episode.
```

Comment:
In support of the recent protests to bring about changes to systemic racial injustice and police brutality [instigated by the murder of George Floyd? Not sure if we want to call out one death when there were so many?], we will will be donating our sponsorship profits from last week's episode to [Campaign Zero](https://www.joincampaignzero.org/) from last week's episode.

Reply:
```suggestion
In support of the protests to bring about changes to systemic racial injustice and police brutality in the United States, we will not be releasing a new episode this week and will be donating our sponsorship profits to [Campaign Zero](https://www.joincampaignzero.org/) from last week's episode.
```

What do you think of this @tessaSac?

Reply:
> To support the US protests against systemic racism and police brutality, we will be donating last week's sponsorship money to Campaign Zero, and refrain from releasing an episode this week in order to focus on the global conversation on these grave injustices.

Might be more in line with the changes to our tweet?